modules/acquia_cms_site_studio/acquia_cms_site_studio.module

```diff
@@ -213,3 +214,20 @@ function acquia_cms_site_studio_rebuild_styles() {
   // rebuild.
   batch_set(install_acms_site_studio_rebuild());
 }
+
+/**
+ * Implements hook_modules_installed().
+ */
+function acquia_cms_site_studio_modules_installed($modules) {
```

Comment:
@rajeshreeputra We shouldn't use this hook as this hooks gets invoked each time a module is installed. For ex: If I install 20 modules one by one, then will be called 20 times, but we don't need to update config 20 times. Please use hook_module_install() and also, add condition check for `node_revision_delete` module only.
cc: @chandan-singh7929

Reply:
updated